config/clusters/2i2c/binderhub-ui-demo.values.yaml

```diff
@@ -40,7 +40,6 @@ jupyterhub:
     redirectToServer: false
     services:
       binder:
-        display: false
```

Comment:
This is relevant when a `url` is specified, but defaults to false whenever there isn't an `url` specified to link out to.